Tests/UnitTests/Plugins/Json/ProtoDetectorJsonConverterTests.cpp

```diff
@@ -0,0 +1,410 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Geometry/ProtoDetector.hpp"
+#include "Acts/Plugins/Json/ActsJson.hpp"
+#include "Acts/Plugins/Json/ProtoDetectorJsonConverter.hpp"
+#include "Acts/Tests/CommonHelpers/FloatComparisons.hpp"
+
+#include <fstream>
+#include <iostream>
+
+using namespace Acts;
+
+BOOST_AUTO_TEST_SUITE(ProtoDetectorJsonConverter)
+
+BOOST_AUTO_TEST_CASE(ProtoDetectorRoundTrip) {
```

Comment:
is this geometry duplicated in the other unit tests? if so it might make sense to deduplicate the construction in a common header

Reply:
It's a different, more complicated geometry, will split it out probably later.